### PR TITLE
Rename the `master` branch to `main`.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

File: libraries/rush-lib/src/api/RushConfiguration.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ const DependencyAnalyzerModule: typeof DependencyAnalyzerModuleType = Import.laz
 );
 
 const MINIMUM_SUPPORTED_RUSH_JSON_VERSION: string = '0.0.0';
-const DEFAULT_BRANCH: string = 'master';
+const DEFAULT_BRANCH: string = 'main';
```
Comment (octogonz) [inline libraries/rush-lib/src/api/RushConfiguration.ts:46]: This should be a MINOR version bump for Rush

File: libraries/rush-lib/src/logic/pnpm/PnpmYamlCommon.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export interface IYamlDumpOptions {
 }
 
 // This is based on PNPM's own configuration:
-// https://github.com/pnpm/pnpm-shrinkwrap/blob/master/src/write.ts
+// https://github.com/pnpm/pnpm-shrinkwrap/blob/main/src/write.ts
```
Comment (octogonz) [inline libraries/rush-lib/src/logic/pnpm/PnpmYamlCommon.ts:12]: This is broken link

File: libraries/rush-lib/src/logic/Git.ts [hunk 3]
```diff
@@ -298,7 +298,7 @@ export class Git {
    * Gets the remote default branch that maps to the provided repository url.
    * This method is used by 'Rush change' to find the default remote branch to compare against.
    * If repository url is not provided or if there is no match, returns the default remote's
-   * default branch 'origin/master'.
+   * default branch 'origin/main'.
```
Comment (octogonz) [inline libraries/rush-lib/src/logic/Git.ts:301]: Where is this implemented?

**Review by octogonz — APPROVED** [review]

File: libraries/rush-lib/src/logic/pnpm/PnpmYamlCommon.ts (hunk 2 above)
Comment (iclanton) [reply libraries/rush-lib/src/logic/pnpm/PnpmYamlCommon.ts:12]:
```suggestion
// https://github.com/pnpm/pnpm-shrinkwrap/blob/master/src/write.ts
```

File: libraries/rush-lib/src/logic/Git.ts (hunk 3 above)
Comment (iclanton) [reply libraries/rush-lib/src/logic/Git.ts:301]: https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/src/api/RushConfiguration.ts#L46

**Pull request merged**